pandas/core/frame.py

```diff
@@ -4648,6 +4653,33 @@ def append(self, other, ignore_index=False, verify_integrity=False):
         2  5  6
         3  7  8
 
+        The following, while not recommended methods for generating DataFrames,
```

Comment:
Hmmm...I'm not sure if you need to demonstrate both methods.  The latter is probably sufficient.  In any case, if you are going to demonstrate this for `DataFrame`, you should do a similar demo for `Series`.